curriculum/challenges/english/25-front-end-development/review-python/67f39e391c9b373069def02c.md

```diff
@@ -0,0 +1,22 @@
+---
+id: 67f39e391c9b373069def02c
+title: Python Review
+challengeType: 24
+dashedName: review-python
+---
+
+# --description--
+
+Review the concepts below to prepare for the upcoming quiz.
```

Comment:
and here. this looks like the python chapter review. instead of quiz.

```suggestion
Review the concepts below to prepare for the upcoming exam.
```